### ISSUE TITLE
Cleanup duplicate entries after renaming a source

### ISSUE THREAD

**Issue by Sanqui** [karlicoss/promnesia#269]
I just started using Promnesia, set up a directory with Markdown files as a source in the config, tested it, then proceeded to rename the source to give it a more descriptive name than `auto`.  I'm now getting duplicate entries in the sidebar, what's the story to clean these up?  Thanks for the awesome tool by the way.

**Comment by karlicoss** [comment]
Thanks, glad you tried it and like it :) 

Good catch, I suspected it could be the case, but never got round to try.

The easiest is perhaps to use `promnesia index --overwrite ...`, this will empty the index database before filling it again, so would get rid of duplicates. I have Promnesia running in overwrite mode once a day so never really end up with dupes.

The reason for default behavior (when it doesn't touch the other sources) is so one could run promnesia on lighter souces frequently. E.g. I'm running it only against my plaintext notes every ten minutes, whereas running in `--overwrite` mode like I mentioned above with all sources would take a while, maybe half an hour.

**Comment by Sanqui** [comment]
Thanks for the tip, that makes sense!  It's good to know the index can be safely emptied like that.

**Comment by karlicoss** [comment]
Yep, the index (by default something like `~/.local/share/promnesia/promnesia.sqlite`) can even be manually removed if necessary -- worst case it'll be just rebuilt next time Promnesia runs, it wouldn't mess with the original data :)
